components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/business-controllers/device.js

```diff
@@ -369,7 +377,9 @@ deviceModule = function () {
 		var carbonUser = session.get(constants["USER_SESSION_KEY"]);
 		if (!carbonUser) {
 			log.error("User object was not found in the session");
-			throw constants["ERRORS"]["USER_NOT_FOUND"];
+            userModule.logout(function () {
```

Comment:
Fix formatting issue.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275